src/osdag/cad/items/double_angles.py

```diff
@@ -0,0 +1,245 @@
+        self.spacing = spacing
+        
+        # Store dimensions for later use
+        self.L = L
+        self.gusset_L = gusset_L
+        self.gusset_H = gusset_H
+        self.gusset_T = gusset_T
+        
+        # Calculate Z-offsets based on angle length
+        # Position plates at the ends with a small margin
+        margin = gusset_L/2  # Adjust this value to fine-tune the end position
+        self.gusset1_offsets = numpy.array([0, 0, margin])  # At the start
+        self.gusset2_offsets = numpy.array([0, 0, L - margin])  # At the end
+        
+        # Base origin and directions
+        self.sec_origin = numpy.array([0, 0, 0])
+        self.uDir = numpy.array([1.0, 0, 0])
+        self.wDir = numpy.array([0.0, 0, 1.0])
+        self.vDir = numpy.cross(self.wDir, self.uDir)
+
+    def place(self, sec_origin, uDir, wDir):
+        """Places the angles and gusset plates with automatic end positioning"""
+        self.sec_origin = sec_origin
+        self.uDir = uDir
+        self.wDir = wDir
+        self.vDir = numpy.cross(self.wDir, self.uDir)
+        
+        # Place angles
+        self.angle1.place(self.sec_origin, self.uDir, self.wDir)
+        
+        origin2 = self.sec_origin + self.spacing * self.vDir
+        rotated_uDir = -self.vDir
+        rotated_vDir = self.uDir
+        self.angle2.place(origin2, rotated_uDir, self.wDir)
+        
+        # Calculate center point between angles
+        center_point = self.sec_origin + (self.spacing / 2) * self.vDir
+        
+        # Place first gusset plate at the start
+        gusset1_origin = (
+            center_point 
+            + self.gusset1_offsets[0] * self.uDir  # X offset (0)
+            + self.gusset1_offsets[1] * self.vDir  # Y offset (0)
+            + self.gusset1_offsets[2] * self.wDir  # Z offset (start position)
+        )
+        gusset1_uDir = numpy.array([0, 0, 1.0])
+        gusset1_wDir = numpy.array([0, -1.0, 0])
+        self.gusset1.place(gusset1_origin, gusset1_uDir, gusset1_wDir)
+        
+        # Place second gusset plate at the end
+        gusset2_origin = (
+            center_point 
+            + self.gusset2_offsets[0] * self.uDir  # X offset (0)
+            + self.gusset2_offsets[1] * self.vDir  # Y offset (0)
+            + self.gusset2_offsets[2] * self.wDir  # Z offset (end position)
+        )
+        gusset2_uDir = numpy.array([0, 0, -1.0])
+        gusset2_wDir = numpy.array([0, 1.0, 0])
+        self.gusset2.place(gusset2_origin, gusset2_uDir, gusset2_wDir)
+
+
+
+    def create_model(self):
+        """Creates the 3D model of back-to-back angles with gusset plates"""
+        # Create models
+        angle1_prism = self.angle1.create_model()
+        angle2_prism = self.angle2.create_model()
+        gusset1_prism = self.gusset1.create_model()
+        gusset2_prism = self.gusset2.create_model()
+        
+        # Combine all shapes
```

Comment:
We probably want something apart from a `BRepAlgoAPI_Fuse`. A look at tension welded/bolted case would be helpful.